Auth.php

```diff
@@ -32,6 +37,7 @@ public function __construct(Model $userModel = null)
         }
 
         $this->userModel = $userModel;
+        $this->fallbackAuth = new \Piwik\Plugins\Login\Auth();
```

Comment:
Just to understand it, another fix would have been to call `parent::__construct` here so the model get's created in `Login/Auth`?